src/internal/views/nunjucks/charge-information/macros/charge-element-table-sroc.njk

```diff
@@ -94,14 +94,14 @@ formErrorSummary %}
       categoryId: categoryId
     } | queryString %}
   {% endif %}
-  {% if chargeElementId %}
-    {{ changeLinkRoot + '/charge-element/' + chargeElementId + urlTail + '?' + queryTail }}
+  {% if chargePurposeId %}
```

Comment:
Is this link correct for charge purpose?

Reply:
yes that is correct will go through the logic with you. not ideal but it is basically re-using the charge element flow for sroc charge purposes which is one and the same thing